lib/daru/view/adapters/googlecharts.rb

```diff
@@ -14,24 +14,32 @@ module GooglechartsAdapter
         # the google charts option concept.
         # and google_visualr : http://googlevisualr.herokuapp.com/
         #
-        # TODO : this docs must be improved
+        # @param data [Array, Daru::DataFrame, Daru::Vector, Daru::View::Table,
+        #   String] The data provided by the user to generate the google chart.
+        #   Data in String format represents the URL of the google spreadsheet
+        #   from which data has to invoked
+        # @param options [Hash] Various options provided by the user to
+        #   incorporate in google charts
+        # @return [GoogleVisualr::Interactive] Returns the chart object based
+        #   on the chart_type
```

Comment:
Thanks for writing this. Please write few examples as well for `init` and `init_table`.